build.gradle.kts

```diff
@@ -138,11 +139,11 @@ tasks.withType<AbstractPublishToMaven> {
 tasks {
 
     compileKotlin {
-        kotlinOptions.jvmTarget = languageLevel
```

Comment:
'kotlinOptions: KotlinJvmOptions' is deprecated. Please migrate to the compilerOptions DSL.